scss/_variables.scss

```
@@ -914,7 +914,7 @@ $dropdown-border-color:             rgba($black, .15) !default;
 $dropdown-border-radius:            $border-radius !default;
 $dropdown-border-width:             $border-width !default;
 $dropdown-inner-border-radius:      subtract($dropdown-border-radius, $dropdown-border-width) !default;
-$dropdown-divider-bg:               $gray-200 !default;
+$dropdown-divider-bg:               $gray-600 !default;
```

Per @ysds's comment, let's roll with the dropdown's border.

```suggestion
$dropdown-divider-bg:               $dropdown-border-color !default;
```

---

Done (separately, as I hadn't even clocked this suggested change commit thing from the email)